### ISSUE TITLE
Ideas for the different Crises

### ISSUE THREAD

**Issue by Jin2188** [Jin2188/CrisisExpansion#1]
Crises do not require a specific ethic, but do require you to NOT have the opposing ethic; for example, a militarist associated crisis would require you to not be pacifist.

Galactic Nemesis: Either militarist or xenophobe associated. Could also be a general crisis accessible to Gestalts as well.
Cosmogenesis: Materialist associated

Devotion Crisis: Spiritualist associated, placeholder name. Focused around using the Devotion resource to manifest beliefs into reality via the Shroud. Ends with you summoning a Shroud deity into reality, with varying effects based on your ethics and civics.

~~Dimensional Tearer: Pacifist or xenophile associated, leaning towards pacifist. Will need Sparble's permission to use the idea, since it originates from his Stellar Hyperconstructs mod. Basically a "safer" version of the Aetherophasic Engine.~~ (probably am not going to do this anymore, tree proposed a much better idea)

Time Manipulator Crisis: Thanks to tree for the idea. Pacifist associated. Focuses on constructing a device that can manipulate past events, defeating your enemies not through conquest, but through erasing important milestones in their development.

Pseudo Hive-Mind Crisis: Thanks to wandering modder for the idea. Authoritarian associated. Focuses on constructing a device that can control the minds of sapients galaxy-wide. Ends with your empire annexing the entire galaxy, and every sapient species in the galaxy gaining a trait that makes them not affected by happiness.

**Comment by TranslatorIV** [comment]
Devotion crisis: Probably manifesting leaders and fleets out of nowhere using devotion.
Summoning deity: Xenophobes and militarists trying to give a deity form and then chain it to use it.(maybe a megastructure.) Spiritualists are given the deity in the form of a leader and go through a chosen one type reformation.

**Comment by Jin2188** [comment]
I like that idea. Imperial Cult could probably end with turning your ruler into a god by merging them with the shroud deity you summon.

**Comment by TranslatorIV** [comment]
Do all ethics require their own crisis? I think the devotion crisis can have results for all ethics shroud entities and that would be enough.

**Comment by Jin2188** [comment]
Devotion crisis isn't spiritualist exclusive, it's just spiritualist associated. You can use any ethic except materialist (because materialists probably won't be so keen on using "blind faith" to do things)

**Comment by TranslatorIV** [comment]
yeah I mean it enough content for a mod already if were gonna have results that differ depending on the empire.

**Comment by Jin2188** [comment]
I mean, you have a point, but I like the idea of each ethic having a crisis associated with them, since we already are getting a very clearly materialist oriented crisis. I always felt like the "Become the Crisis" system was too basic, forcing you onto a single, narrow path. Cosmogenesis mitigates this a little bit, but I feel like adding several new crisis paths will help a lot with adding replayability to the system and make it more unique.

I mean after all, we're only changing the outcome of the devotion crisis based on the empire, the actual path to get there and the general goal remain the same.

**Comment by Jin2188** [comment]
Maybe the Authoritarian crisis can be kinda what you mentioned in Discord; sort of a "psi-emitter" thingie that controls people. The elite will maintain their free will, and will be able to control the masses to do their bidding; a sort of "pseudo hive mind".

I was at a loss for what to do with Egalitarian, but I'm kinda thinking of a sort of anarchist revolution that you can spread to other empires through war and espionage.

**Comment by TranslatorIV** [comment]
Egalitarian and pacifists can just force others into accepting their ideology. As a crisis they will probably be the hardest cause there's no magic win the game now button.

**Comment by Jin2188** [comment]
For Egalitarian, it could be kind of like what Marx envisioned communism would be. Either through war or espionage, you cause other empires' governments to collapse, with each of their planets becoming individual communes; no central planetary government, just people working together. Once you've done this to every empire in the galaxy, you can do it to yourself, and "win" the game by dissolving your own government.

**Comment by TreeM4n** [comment]
All my ideas are already mentioned. My only reamaining idea is a perk for deity summon: whenever a fleet with a commander enters a fight summon an allied follower of said deity (grows stronger the higher level you are), maybe even an avatar for ground combats

> Egalitarian and pacifists can just force others into accepting their ideology. 

> For Egalitarian, it could be kind of like what Marx envisioned communism would be. Either through war or espionage, you cause other empires' governments to collapse, with each of their planets becoming individual communes; no central planetary government, just people working together. Once you've done this to every empire in the galaxy, you can do it to yourself, and "win" the game by dissolving your own government.

Strongly against both. Crisis paths are a threat to the galaxy, not some failed ideology. Egalitarian and pacifists don't usually want to be a threat. Their play style is the space UN and galactic empire (ironically changes you to authorial). 

Pacifists are way too harmless to something like a crisis, but what if they try to interfere with others weapons on a galaxy level? Like giving every planet shields, reducing your weapon damage. They could use a megastructure to alter the flow of history. Totally stripping empires of their military potential (as they never developed it). **Kinda a time machine crisis**, which regresses offensive potential galaxy wide.

**Comment by Jin2188** [comment]
> Crisis paths are a threat to the galaxy, not some failed ideology.

This is a fair point. I just couldn't really think of any other way to do an Egalitarian crisis. I want it to have something, maybe we could add some esoteric technology to the idea.

> Kinda a time machine crisis, which regresses offensive potential galaxy wide.

Now this is a cool idea. I'm imagining it in the same vein as Doctor Who, except instead of one person, it's an entire empire. I wonder if it's possible to make empires "un-research" technologies. Is there an effect for that?

**Comment by Jin2188** [comment]
I've written up a draft for some basic core ideas for the other crises. I know we're supposed to build upon our other ideas first, but I wanted to get a basic idea of the "crisis spectrum" for all the ethics. I still have no idea what to do for Egalitarian, but I do have a core theme for it here:

[Other Crisis Ideas Basic Draft.docx](https://github.com/Jin2188/CrisisExpansion/files/15102133/Other.Crisis.Ideas.Basic.Draft.docx)

**Comment by TranslatorIV** [comment]
I'm against the become the evil to unite against idea. We are talking about entire empires, not individuals. Convincing an entire nation that we're gonna be the bad guys on purpose even though we lose more than we gain does not make sense. A galaxy wide indoctrination system might make more sense. For egalitarians and xenophiles it could "suggest" equality for all. It would be enough of a threat for other empires to try and stop you, but also not require you to become a warmongering faction.

**Comment by Jin2188** [comment]
The Galactic Nemesis has no problem convincing its population to commit genocide, even if they're not militarist or xenophobe. Plenty of nations in our own world have been able to convince their people to assist in things that do not benefit them in the slightest, as long as it's for a "greater good". I honestly think it makes perfect sense for a warped pacifistic/xenophilic mentality.

I'm against the idea of a galactic indoctrination for the same reason tree mentioned earlier:

> Crisis paths are a threat to the galaxy, not some failed ideology.

The crisis paths need to actually be dangerous beyond just making people align with you. Also a galactic indoctrination is a bit too similar to the authoritarian crisis we already have an idea for. I like the idea otherwise, but it just doesn't seem to meet the "existential threat" qualifications for a crisis.
